wazimap_ng/points/migrate_tooltip_data.py

```diff
@@ -0,0 +1,17 @@
+
+
+from wazimap_ng.points.models import ProfileCategory
+
+def migrate_attributes():
```

Comment:
I meant a data **migration**

Create a new migration file and update the field there. 

it should look like this:
https://github.com/OpenUpSA/wazimap-ng/compare/chore/data-migration#diff-b5add147a76fbb22069454c268cf5e84368d0f14b12e921b3891192440ef50c6